lib/health_check.rb

```diff
@@ -0,0 +1,91 @@
+class HealthCheck
+  include ChefResource
+
+  REACHABLE = 'reachable'.freeze
+  TIMEOUT = 'timeout'.freeze
+  UNREACHABLE = 'unreachable'.freeze
+  ERRORING = 'erroring'.freeze
+  KEYERROR = 'key error'.freeze
+
+  attr_reader :status, :erchef, :postgres
+
+  def initialize
+    @status = nil
```

Comment:
Since we're initializing erchef and postgres to reachable, perhaps status should be initialized to ok. Might also help us avoid hitting a nil pointer error at some point too. And perhaps the status values of ok and not ok should also be constants?